main.css

```diff
@@ -1,166 +1,640 @@
 .navigation-container {
-    align-self: center;
-    display: flex;
-    width: 100%;
-    max-width: 1400px;
-    align-items: center;
-    gap: 20px 100px;
-    justify-content: space-between;
-    flex-wrap: wrap;
-  }
-  
-  @media (max-width: 1200px) {
-    .navigation-container {
-      max-width: 100%;
-    }
-  }
-  
-  .brand-title {
-    color: var(--Brand-600, #25a59e);
-    -webkit-text-stroke: 0.7px var(--Brand-600, #25a59e);
-    letter-spacing: 2.8px;
-    align-self: stretch;
-    margin: auto 0;
-    font: 400 28px Gilda Display, sans-serif;
-  }
-.travel-heading {
-  color: var(--Neutral-900, #262626);
-  text-align: center;
   align-self: center;
-  margin-top: 135px;
-  font: 400 170px/170px Gilda Display, -apple-system, Roboto, Helvetica, sans-serif;
+  display: flex;
+  width: 100%;
+  max-width: 1400px;
+  align-items: center;
+  gap: 20px 100px;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+@media (max-width: 1200px) {
```

Comment:
# For best practice
- Scattering media queries throughout your CSS can lead to disorganized and hard-to-maintain code
- Group media queries by screen size or breakpoint at the end of your stylesheet